docs/Reference/API-Methods.md

```diff
@@ -4205,6 +4205,72 @@ data using `eea_sendRawTransaction`.
     The `PRIV` API methods are not enabled by default for JSON-RPC. Use the [`--rpc-http-api`](CLI/CLI-Syntax.md#rpc-http-api)
     or [`--rpc-ws-api`](CLI/CLI-Syntax.md#rpc-ws-api) options to enable the `PRIV` API methods.
 
+### priv_call
+
+Invokes a private contract function locally and does not change the state of the blockchain.
```

Comment:
s/the state of the blockchain/privacy groups state/

Reply:
Fixed